modules/scheduler/scheduler/src/main/java/org/motechproject/scheduler/contract/JobDto.java

```diff
@@ -0,0 +1,69 @@
+package org.motechproject.scheduler.contract;
+
+import org.codehaus.jackson.map.annotate.JsonDeserialize;
+import org.codehaus.jackson.map.annotate.JsonSerialize;
+import org.joda.time.DateTime;
+import org.motechproject.scheduler.util.CustomDateDeserializer;
+import org.motechproject.scheduler.util.CustomDateSerializer;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class JobDto {
+
+    private JobType type;
+
+    private String motechEventSubject;
+
+    private Map<String, Object> motechEventParameters;
+
+    @JsonDeserialize(using = CustomDateDeserializer.class)
```

Comment:
why do we need custom date serializing/deserializing? We manage without in other places